### PR TITLE
edits

### PR DESCRIPTION
**Pull request by Leslie-Evangelista**
pls look

### CODE REVIEW

File: TicTacXOs/src/styles/components.css [hunk 1]
```diff
@@ -98,4 +106,60 @@
...
+  border: none;
+  border-radius: 15px;
+  width: 223px;
+  height: 81px;
+  color: #FFFFFF;
+  font-family: 'Baloo', cursive;
+  font-size: 30px;
+}
+
+#green-button {
```
Comment (raquel-castoldi) [inline TicTacXOs/src/styles/components.css:133]: these should be . instead of #

File: TicTacXOs/src/styles/components.css [hunk 2]
```diff
@@ -1,3 +1,9 @@
+#logo {
+  padding: 20px 40px 20px 40px;
+  width: 300px;
+}
+
+/* Game bored*/
```
Comment (raquel-castoldi) [inline TicTacXOs/src/styles/components.css:6]: misspelling of board

File: TicTacXOs/src/styles/components.css (hunk 1 above)
Comment (raquel-castoldi) [inline TicTacXOs/src/styles/components.css:133]: . instead of #

File: TicTacXOs/src/styles/components.css [hunk 3]
```diff
@@ -98,4 +106,60 @@
...
+  font-size: 30px;
+}
+
+#green-button {
+
+  background-color: #39D87C;
+
+}
+
+#yellow-button {
```
Comment (raquel-castoldi) [inline TicTacXOs/src/styles/components.css:139]: . instead of #

File: TicTacXOs/src/styles/components.css [hunk 4]
```diff
@@ -98,4 +106,60 @@
...
+
+  background-color: #CFCF5B;
+
+}
+
+#red-button {
```
Comment (raquel-castoldi) [inline TicTacXOs/src/styles/components.css:145]: . instead of #

File: TicTacXOs/src/styles/components.css [hunk 5]
```diff
@@ -98,4 +106,60 @@
...
+
+  background-color: #F64444;
+
+}
+
+#instructions {
```
Comment (raquel-castoldi) [inline TicTacXOs/src/styles/components.css:151]: this has to be changed according to the current code

File: TicTacXOs/src/styles/components.css [hunk 6]
```diff
@@ -98,4 +106,60 @@
     color: white;
     width: 100%;
     font-size: 1.5em;
-  }
\ No newline at end of file
+  }
+
+/* End of game bored*/
```
Comment (raquel-castoldi) [inline TicTacXOs/src/styles/components.css:111]: mispelled

File: TicTacXOs/src/styles/components.css [hunk 7]
```diff
@@ -1,3 +1,9 @@
+#logo {
```
Comment (raquel-castoldi) [inline TicTacXOs/src/styles/components.css:1]: this was for a pic of the logo i used in the static website, has to be changed bc now it is text not a photo
font info is on figma

**Review by raquel-castoldi — CHANGES_REQUESTED** [review]

File: TicTacXOs/src/styles/components.css (hunk 7 above)
Comment (raquel-castoldi) [reply TicTacXOs/src/styles/components.css:1]: also you should add a font categoru

**Review by Leslie-Evangelista — COMMENTED** [review]
please again

**Pull request merged**